openassistant/templates/template.py

```diff
@@ -36,10 +36,10 @@
 
 import datasets
 
-from .hub import OpenAssistantConfig
+from hub import OpenAssistantConfig
 
 # TODO: import the schema (i.e. features) that fits your dataset:
-from .hub import
+from hub import
```

Comment:
I've added an `__init__.py` in main. could you merge and check whether this change is still necessary?